tests/test_phase5_rebuild_ete.py

```
@@ -0,0 +1,648 @@
+    )
+    assert view["kind"] == "line", f"in-corner kind must be 'line', got {view['kind']}"
+
+
+# ---------------------------------------------------------------------------
+# ETE-05: Manifest contract — FIXED_SIZE + PADDING=0,0
+# ---------------------------------------------------------------------------
+
+
+def test_ete05_manifest_contract():
+    """ETE-05: manifest.ini WINDOW_0 and WINDOW_1 have FIXED_SIZE + PADDING=0,0
+    + reasonable defaults. The user's 132×456 saved geometry is recovered by
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)